common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -1533,10 +1534,19 @@ def test_single_demand_hint(self):
         self.assertFalse(result['should_enable_next_hint'])
 
     def test_demand_hint_logging(self):
+        def mock_location_text(self):
+            """
+            Mock implementation of __unicode__ or __str__ for the module's location.
+            """
+            return u'i4x://edX/capa_test/problem/meh'
+
         module = CapaFactory.create(xml=self.demand_xml)
         # Re-mock the module_id to a fixed string, so we can check the logging
         module.location = Mock(module.location)
-        module.location.to_deprecated_string.return_value = 'i4x://edX/capa_test/problem/meh'
+        if six.PY2:
```

Comment:
Is it too early to start putting in TODO/SHIM comments for Py3?

Reply:
I think an argument can be made that all we have to do for that is look for places where we use `six` (and there will be thousands of those by the time we finish the upgrade)